### ISSUE TITLE
Information please ?

### ISSUE THREAD

**Issue by spordiab** [jnthas/clockwise#71]
hello first of all I wanted to thank you for your work.  I would like to understand if the poster is normal or there is a bug?  The PacMan clock turns red from time to time.  THANKS
![2024-05-16-10-34-46-027](https://github.com/jnthas/clockwise/assets/168900799/90a25e58-062a-49c0-a84a-518dddfb88e3)

**Comment by jnthas** [comment]
Hi @spordiab! It's looks there is some misconnection. Please review your wiring again. Half of display is red, probably some section A,B,C,D,E in the display is wrong.

**Comment by Kalki2898add** [comment]
Can i use 2 p5 32 x 64 display connect series. Will it work ?

**Comment by jnthas** [comment]
No, only 64x64 ones are supported. I'm closing this issue, feel free to re-open if necessary
